src/stdfblib/ita/OPCUA_DEV.h

```diff
@@ -24,5 +24,7 @@ class OPCUA_DEV : public RMT_DEV {
   OPCUA_DEV();
   virtual ~OPCUA_DEV();
 
+  static std::unique_ptr<CDevice> createDev(const std::string &paMGRID);
```

Comment:
this too should be removed

Reply:
done.

Just for completness: OPCUA_DEV will in addition need a major rework. But that is something for another PR.